html/class_passwordRecovery.inc

```diff
@@ -147,10 +147,15 @@ class standAlonePage {
     clean_smarty_compile_dir($smarty->compile_dir);
 
     $smarty->assign('password_img', 'geticon.php?context=status&icon=dialog-password&size=48');
+    $smarty->assign('mail_img', 'geticon.php?context=applications&icon=internet-mail&size=48');
+    $smarty->assign('directory_img', "geticon.php?context=places&icon=network-server&size=48");
+    $smarty->assign('personal_img', "geticon.php?context=types&icon=user&size=48");
```

Comment:
Not sure of the benefit of putting that here, maybe directly put images paths in the tpl file?